docs/src/using-criticalup/toolchain-management.rst

```diff
@@ -120,3 +119,19 @@ From the direcory containing the relevant ``criticalup.toml``:
 .. code-block::
 
    criticalup verify
+
+Creating Archives of Toolchains
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CriticalUp can produce uncompressed tarballs of toolchains which can then be
```

Comment:
```suggestion
CriticalUp can produce uncompressed archive files of toolchains which can then be
```

Reply:
I think let's leave this one so users know it makes a tar. If we add other formats etc in the future, we'll have to update this?

Reply:
Sounds good.